x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/components/actions_menu.tsx

```diff
@@ -85,7 +85,7 @@ export const AgentConfigActionMenu = memo<{
                   >
                     <FormattedMessage
                       id="xpack.ingestManager.agentConfigActionMenu.enrollAgentActionText"
-                      defaultMessage="Enroll agent"
+                      defaultMessage="Add Agent"
```

Comment:
I think it should be `Add agent` (lowercase agent) throughout?

Reply:
I think we use Agent upper case in our doc like here for example https://www.elastic.co/downloads/elastic-agent

Reply:
when it is the full "Elastic Agent" term we capitalize but I'm not sure about our UI when we have phrases like "your agents" or "View agents". maybe @dedemorton or @hbharding can clarify

Reply:
I agree with Jen. Since we're not prefacing it with "Elastic", it should be lowercase "agent". We use lower case "agent" throughout the UI. I think this is something we should revisit in 7.10 (maybe we want to say "Add Elastic Agent"), but for now, let's stay consistent and just do "Add agent"

Reply:
I didn't get to comment before this was merged, but I prefer lowercase. The style guide currently says to capitalize Agent (hence it's capitalized in the docs), but I've initiated an issue to get that changed: https://github.com/elastic/style-guide/issues/176.